batchflow/models/torch/batch_opt_mixin.py

```diff
@@ -0,0 +1,336 @@
+""" Contains mixin for :class:`~.torch.TorchModel` to provide textual and graphical visualizations. """
```

Comment:
Its not

Reply:
Also, I would rather name this file `base_batchsize_mixin.py`

Reply:
What's wrong with `batch_opt_mixin.py`? I think it's more self-explanatory

Reply:
All of the mixins for `TorchModel` are now in `base_*.py` files: that is the main reason

Reply:
Fair enough